javascript/javascript1/week3/Series duration of my life.js

```diff
@@ -0,0 +1,43 @@
+const seriesDurations = [
+ {
+     title: "Matador",
+     days: 1,
+     hours: 0,
+     minutes: 0,
+ },
+ {
+     title: "the wizard",
+     days: 3,
+     hours: 0,
+     minutes:10,
+ },
+ {
+     title: "you",
+     days: 2,
+     hours: 5,
+     minutes: 10,
+ },
+];
+
+ //my age 80
+lifespan = 80 * 365 ;
+// how much take time of my life each of series
+function percentage(array) {
+ let sum = 0
+ for (i = 0; i < array.length; i++){
```

Comment:
Hej shahnaz. it is very hard work you did , just small thing in for loop we have to write type of variable ,let or const.